src/Microsoft.DotNet.InternalAbstractions/Native/NativeMethods.Unix.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.InternalAbstractions.Native
+{
+    internal static partial class NativeMethods
+    {
+        public static class Unix
+        {
+            public unsafe static string GetUname()
```

Comment:
This method only appears to be called from #if NET451.  If we need to keep it, can it just be compiled for NET451?